pkg/client/rp/relying_party.go

```diff
@@ -154,6 +154,8 @@ func NewRelyingPartyOAuth(config *oauth2.Config, options ...Option) (RelyingPart
 		}
 	}
 
+	_ = rp.IDTokenVerifier()
```

Comment:
is it only a problem for the OAuth RP?

Reply:
I'm not sure I understand the question.   It's a problem if an id token needs to get verified...

Reply:
The "precalls" to the methods or only done in the OAuth provider constructor but not in the OIDC (discovery based) one. If i get this right, this would need them as well:
<img width="597" alt="image" src="https://user-images.githubusercontent.com/9405495/193507961-72e44371-a019-4ed7-a8d4-0614366f4517.png">

Reply:
Oh!   Got it.  Thanks.   I'll add that to the PR.   I had been blind to the OAuth constructor.